packages/webapp/src/_app/config.tsx

```diff
@@ -0,0 +1,68 @@
+import { IoHomeOutline } from "react-icons/io5";
+import { FiCheckSquare, FiUsers } from "react-icons/fi";
+import { AiOutlineIdcard } from "react-icons/ai";
+import { BsStar } from "react-icons/bs";
+
+export interface Route {
+    href: string;
+    label: string;
+    exactPath?: boolean;
+    hideNav?: boolean;
+    NavIcon?: React.ComponentType;
+}
+
+export const ROUTES: { [key: string]: Route } = {
+    HOME: {
+        href: "/",
+        label: "Home",
+        NavIcon: () => <IoHomeOutline className="text-2xl xl:text-xl" />,
+        exactPath: true,
+    },
+    MEMBERS: {
+        href: "/members",
+        label: "Community",
+        NavIcon: () => <FiUsers className="text-2xl xl:text-xl" />,
+    },
+    INDUCTION: {
+        href: "/induction",
+        label: "Membership",
+        NavIcon: () => (
+            <AiOutlineIdcard
+                className="text-3xl xl:text-2xl -ml-px -mr-px -mt-1"
+                style={{ marginBottom: -2, marginTop: -2 }}
+            />
+        ),
+    },
+    DELEGATION: {
+        href: "/delegates",
+        label: "My Delegates",
+        hideNav: true,
+        NavIcon: () => (
+            <BsStar
+                className="text-3xl xl:text-2xl -ml-px"
+                style={{ marginBottom: -2, marginTop: -2 }}
+            />
+        ),
+    },
+    TREASURY: {
+        href: "/treasury",
+        label: "Treasury",
+        NavIcon: () => (
+            <img
+                src="/images/eos-logo.svg"
+                alt="EOS logo"
+                className="h-7 mx-0.5"
+            />
+        ),
+    },
+    ELECTION: {
+        href: "/election",
+        label: "Election",
+        NavIcon: () => (
+            <FiCheckSquare
+                className="text-3xl xl:text-2xl -ml-px -mt-px -mb-px"
+                style={{ fontSize: 28 }}
+            />
+        ),
+    },
+};
```

Comment:
thought: I don't like the idea of mixing react components with plain configs... When I imagine configs I think about declaring stuff... so the old router could be extended as:
```ts
{ 
  INDUCTION: {
    href: "/induction",
    label: "Membership",
    icon: AiOutlineIdCard,
    size: "regular",
  }
}
```

I see that we have some where they have images, so the interface could offer icon or image as optional... And yeah, there are some variations  of styles and padding, but I think we could wrap them in parameters too.

I think we are totally fine for now but we should probably revisit at some point.

Reply:
Yeah, agreed. I felt some dissonance here too. I'll lodge an issue for us to revisit this.

Reply:
Fixed. The problem I was working around was obviated by some new SVGs Thomas just sent me. 🤘